Core/Core/Constants/UserAgent.swift

```diff
@@ -45,6 +46,8 @@ public enum UserAgent: CustomStringConvertible {
             let version = UIDevice.current.systemVersion
             let os = version.replacingOccurrences(of: ".", with: "_")
             return "Mozilla/5.0 (iPhone; CPU iPhone OS \(os) like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/\(version) Mobile/15E148 Safari/604.1"
+        case .desktopSafari:
```

Comment:
nice 💪, i like having all our userAgents in 1 spot, we used to have them all over the place